### ISSUE TITLE
working?

### ISSUE THREAD

**Issue by victorrpf** [abudimirov/cita-check-spain#1]
Is this bot working?

**Comment by abudimirov** [comment]
Yes, it was working last I checked 3 weeks ago. But be careful, do not run it too often. Use some kind of timeout between checks, otherwise you could be banned. How to run it locally is described on readme
